### PR TITLE
issue #484 include first empty rows and columns in range if skip_null false

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

**Review by BadSingleton — APPROVED** [review]
lgtm

File: include/xlnt/worksheet/worksheet.hpp [hunk 1]
```diff
@@ -433,9 +433,11 @@ class XLNT_API worksheet
     column_t highest_column_or_props() const;
 
     /// <summary>
-    /// Returns a range_reference pointing to the full range of non-empty cells in the worksheet.
+    /// Returns a range_reference pointing to the full range of cells in the worksheet.
+    /// If skip_null is true (default), empty cells (For example if the first row or column is empty)
```
Comment (benoithudson) [inline include/xlnt/worksheet/worksheet.hpp:437]: maybe `skip_empty`?

Comment (vkovec) [reply include/xlnt/worksheet/worksheet.hpp:437]: sure, I was using the same naming convention as the caller (rows() and columns())

Comment (benoithudson) [reply include/xlnt/worksheet/worksheet.hpp:437]: nevermind I see you're copying the name

**Review by benoithudson — APPROVED** [review]

**Pull request closed without merging**